filebeat/docs/modules/aws.asciidoc

```diff
@@ -15,8 +15,8 @@ This file is generated! See scripts/docs_collector.py
 beta[]
 
 This is a module for aws logs. It uses filebeat s3 input to get log files from
-AWS S3 buckets with SQS notification. This module supports reading s3 server
-access logs with `s3access` fileset, ELB access logs with `elb` fileset, VPC
+AWS S3 buckets with SQS notification or directly polling list of S3 objects in an S3 bucket.
+This module supports reading s3 server access logs with `s3access` fileset, ELB access logs with `elb` fileset, VPC
 flow logs with `vpcflow` fileset, and CloudTrail logs with `cloudtrail` fileset.
 
```

Comment:
I think the docs should explain trade-offs and use cases for the two "modes". Like steer users towards SQS S3 notifications primarily and then suggest S3 listing for back-filling existing logs.

Reply:
Oki. Actually there are cases where the SQS S3 notifications could not be setup at all, that was the main reason for introducing a direct S3 bucket polling.